tests/Http/Role/ReadRoleRestControllerTest.php

```diff
@@ -296,7 +296,8 @@ public function it_returns_an_array_of_permissions_for_the_current_god_user()
             "GEBRUIKERS_BEHEREN",
             "LABELS_BEHEREN",
             "MEDIA_UPLOADEN",
-            'VOORZIENINGEN_BEWERKEN'
+            'VOORZIENINGEN_BEWERKEN',
+            'PRODUCTIES_AANMAKEN'
```

Comment:
```suggestion
            'PRODUCTIES_AANMAKEN',
```
(This used to be a coding standards check but it seems it was removed a while back)